lib/mini_autobot/settings.rb

```diff
@@ -93,6 +93,14 @@ def verbosity_level
       hsh.fetch(:verbosity_level, 0).to_i
     end
 
+    def google_sheets?
+      true unless hsh[:google_sheets] == false
```

Comment:
This returns `nil` when `hsh[:google_sheets]` is defined. Since this is a predicate method (a method ending in `?`), you should stick to returning a boolean by convention. Since a `#==` returns a boolean, you can just:

```ruby
def google_sheets?
  !hsh[:google_sheets]
end
```